File: Data Structures & Algorithms/is-palindrome/submission-1.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string p="";
        string r="";
        for(int i=0;i<s.size();i++){
            char c=s[i];
            if(c>='A'&&c<='Z')c=c-'A'+'a';
            if((c>='a'&&c<='z')||(c>='0'&&c<='9')){
                p=p+c;
                r=c+r;
            }
        }
        return p==r;
    }
};

File: Data Structures & Algorithms/string-encode-and-decode/submission-1.py
class Solution:

    def encode(self, strs: List[str]) -> str:
        ans=""
        for st in strs:
            ans=ans+' /'+st
        return ans
    def decode(self, s: str) -> List[str]:
        ans=s.split(' /')
        ans.remove("")
        return ans
